Dockerfile

```diff
@@ -147,6 +152,10 @@ RUN curl -LO \
     && ln -s /usr/local/cerebro-${CEREBRO_VERSION} /usr/local/cerebro \
     && rm -rf /tmp/cerebro*
 
+# ES plugins
+RUN /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch repository-s3
+RUN echo $AWS_ACCESS_KEY | /usr/share/elasticsearch/bin/elasticsearch-keystore add --stdin s3.client.default.access_key
+RUN echo $AWS_SECRET_KEY | /usr/share/elasticsearch/bin/elasticsearch-keystore add --stdin s3.client.default.secret_key
```

Comment:
@remoteeng00 because we use this `Dockerfile` to build multiple varieties of biothings hub/studio image, not just for demohub, see `Makefile`. So let's keep the default behavior unchanged, you can either:

* Use an ARG variable to controle the demohub specific installs
* Create a separate Dockerfile under demohub folder, leave this one unchanged

This applies for all your changes below to setup a FTP server.